mist-api/src/routes/genes/get.js

```diff
@@ -1,17 +1,41 @@
 'use strict'
 
+// Local
+const searchUtil = require('lib/util')
+
 module.exports = function(app, middlewares, routeMiddlewares) {
 	let models = app.get('models'),
 		helper = app.get('lib').RouteHelper.for(models.Gene)
 
+	const exactMatchFieldNames = [
+			'version',
+			'locus',
+			'old_locus',
+			'stable_id'
+	]
+	const textFieldNames = ['product']
+
 	return [
 		middlewares.parseCriteriaForMany(models.Gene, {
 			accessibleModels: [
+				models.Genome,
 				models.Component,
 				models.Aseq,
 				models.Dseq
-			]
+			],
+			permittedWhereFields: [
+				'id'
```

Comment:
You are right this is not searchable by default. Perhaps we should make the route `GET /genomes/:value` search the id field automatically if it is comprised purely of numbers. That would make things a bit simpler and make the following equivalent:
`GET /genes/$id`
`GET /genes/$accession`